Dockerfile

```diff
@@ -25,15 +25,14 @@ COPY Gemfile Gemfile.lock /source/
 
 RUN apk --no-cache add \
         build-base \
+        libstdc++ \
```

Comment:
I'm curious what (newly) depends on this

Reply:
libsaas, as it turns out.